tests/test_servicebus.py

```diff
@@ -79,7 +91,7 @@ def test_servicebus_metadata(self):
 
         # Datetime should be in iso8601 string instead of datetime object
         metadata_dict = servicebus_msg.metadata
-        self.assertDictEqual(metadata_dict, {
+        self.assertGreaterEqual(metadata_dict.items(), {
```

Comment:
Does this compare all the metadata items? If yes, then why the "Greater"?

Reply:
This test is amended to ensure the metadata_dict has the following expected fields (**DeliveryCount**, **LockToken**, **ExpiresAtUtc**...).

Using **self.assertDictEqual** is too strict for this test scenario. We just want to ensure the expected fields are a subset of the metadata. I tried using **self.assertDictContainsSubset** but it has been deprecated in Python 3.6. So I first convert the dictionary into an ItemView using **.items()**, and use **self.assertGreaterEqual** for subset check.